webpack.config.web.js

```diff
@@ -0,0 +1,6 @@
+const config = require("webpack-config-narazaka-ts-js").web;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->